schema.json

```diff
@@ -653,7 +674,172 @@
                             "description": "The social engagement with the advert, presented as an array (likes/shares/comments if these features are available)",
                             "type": "object"
                         }
-                    }
+                    },
+                    "allOf": [
+                        {
+                            "if": {
+                                "properties": {
+                                    "advertising_platform": {
+                                        "oneOf": [
+                                            {
+                                                "properties": {
+                                                    "platform_name": {
+                                                        "const": "Facebook"
+                                                    }
+                                                },
+                                                "required": [
+                                                    "platform_name"
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                }
+                            },
+                            "then": {
+                                "properties": {
+                                    "engagement_social": {
+                                        "type": "object",
+                                        "properties": {
+                                            "likes": {
```

Comment:
these aren't only in english are they

Reply:
That is correct. I have only defined the English ones and allowed the object to be extensible so the other languages will be accepted as well